Tests/StreamChatTests/StreamChatFlakyTests.xctestplan

```diff
@@ -9,7 +9,8 @@
     }
   ],
   "defaultOptions" : {
-
+    "defaultTestExecutionTimeAllowance" : 60,
+    "testTimeoutsEnabled" : true
```

Comment:
60 is the lowest value Xcode allows to set. This is used for failing async test cases which never return